src/reaper_osara.cpp

```diff
@@ -1176,11 +1176,17 @@ void clickIoButton(MediaTrack* track, bool rightClick=false) {
 
 bool maybeSwitchToFxPluginWindow() {
 	HWND window = GetForegroundWindow();
-	char name[4];
+	char name[8];
 	if (GetWindowText(window, name, sizeof(name)) == 0)
 		return false;
-	if (strncmp(name, "FX: ", 4) != 0)
+	if (strncmp(name, "FX: ", 4) != 0 &&
+		strncmp(name, "DX: ", 4) != 0 &&
+		strncmp(name, "VST: ", 5) != 0 &&
+		strncmp(name, "VSTi: ", 6) != 0 &&
+		strncmp(name, "VST3: ", 6) != 0 &&
+		strncmp(name, "VST3i: ", 7) != 0) {
```

Comment:
nit: These if condition lines are indented to the same level as the code inside the if block. Please either indent them an extra level:

```
	if (strncmp(name, "FX: ", 4) != 0 &&
			...
			strncmp(name, "VST3i: ", 7) != 0) {
```

or keep them as they are but have the closing brackets at the same level as the if:

```
	if (strncmp(name, "FX: ", 4) != 0 &&
		...
		strncmp(name, "VST3i: ", 7) != 0
	) {
```

Reply:
Aligned.